server/lib/utils.js

```diff
@@ -197,6 +197,22 @@ const defaultHostId = () => {
 }
 
 /**
+ * Check if this is an internal email address.
+ * Useful for testing emails in localhost or staging
+ */
+const isEmailInternal = (email) => {
+  if (email.toLowerCase().indexOf('@opencollective.com') != -1 ||
+    email.toLowerCase().indexOf('@opencollective.org') != -1 ||
+    email.toLowerCase() === 'asood123@gmail.com' ||
+    email.toLowerCase() === 'piamancini@gmail.com' ||
+    email.toLowerCase() === 'xdamman@gmail.com' ||
+    email.toLowerCase() === 'sedubois@gmail.com') {
```

Comment:
was a bit torn about it. 

Actually, have an idea. I'll wipe the recipient if it's not internal and send the email to the bcc address. That way, we can remove our personal addresses.